__tests__/app.test.js

```diff
@@ -191,8 +196,49 @@ describe('/api/articles/:article_id/comments', () => {
             expect(body.msg).toBe('400 Bad Request: Invalid Id')
         })
     })
-})
+    test('POST 201: responds with the a named object containing the posted comment', () => {
+        const input = {
+            //Presumably the user would have to have an account before posting, so i used an already existing user
+            //Please let me know if this is correct in the feedback!
+            username: 'lurker',
+            body: 'Cool article!'
+        }
+
+        return request(app)
+        .post('/api/articles/1/comments')
+        .send(input)
+        .expect(201)
+        .then(({body}) => {
+            expect(body.comment).toEqual({author: 'lurker', body: 'Cool article!'})
+            expect(body.comment).toBeObject()
+        })
+    })
```

Comment:
Please test if unnecessary properties in the post object are ignored, which means that as long as there is a username and body, the post is successful, no matter how many extra properties are there in the input object.